### DISCUSSION TITLE
Sleeky plugin : remove "view info & stats"

### DISCUSSION THREAD

**Discussion by lazycan** [YOURLS/YOURLS#3216] [Plugins]
Hello, 
I use the Sleeky plugin and I want to remove the sentence "view info & stats" which appears when the shortened link is created.
Is it possible ? In which file ? Thank you.
![shortened link](https://user-images.githubusercontent.com/55274052/152211865-96710ee0-299e-48c4-b1a8-6d129da57fdf.jpg)

**Comment by dgw** [comment]
You should ask the developer(s) of that plugin.
